data/build-flavors/aws.toml

```diff
@@ -0,0 +1,59 @@
+# Generic (aka "universal") ISO image + AWS STUFF (to be able to create an AMI)
+
+image_format = "iso"
+
+packages = [
+  "qemu-guest-agent",
+  "cloud-init",
+  "awscli",
+]
+
+[[includes_chroot]]
+  path = "etc/cloud/cloud.cfg.d/90_dpkg.cfg"
+  data = '''
+datasource_list: [ Ec2 ]
```

Comment:
It is better to add `None` in the end on the list.

Reply:
done